jupyterlab/jupyter_notebook_config.json

```diff
@@ -6,5 +6,8 @@
     "nbserver_extensions": {
       "js9server.js9ext": true
     }
+  },
+  "KernelSpecManager": {
+    "whitelist": set("lsst")
```

Comment:
You know, I don't think this does anything, because I took the sledgehammer approach of removing the "system" kernel too.  I will try taking it out and make sure that it breaks nothing.  This is in fact eventually executed by jupyter as python code, I think.

The shell isn't in rubinenv until you source /opt/lsst/software/stack/loadLSST.bash -- but there's no python 3 or gcc or, well, much in the shell until you do that.